pkg/resource_deployment_test.go

```diff
@@ -189,3 +224,43 @@ func TestExpandDeploymentOverrideProjectID(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, depl, expandedDepl)
 }
+
+func testDeploymentConfig(resource, name, project string) string {
+	return fmt.Sprintf(`resource "oasis_deployment" "%s" {
+	terms_and_conditions_accepted = "true"
+	name        = "%s"
+	description = "Terraform Generated Deployment"
+	project     = "%s"
+	location {
+	  region = "gcp-europe-west4"
+	}
+	version {
+	  db_version = "3.6.0"
```

Comment:
That seems too old.
How can we keep this current?
Perhaps envvar?

Reply:
3.6.0 is available for _support, so OK for internal testing.

Reply:
I wondered why the version is required, if we make it optional, we can simple use the default version?

Reply:
as discussed made version optional (and defaults to the current)